Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -63,6 +63,7 @@ class BrowserLocationView: UIView {
 
     lazy var urlTextField: UITextField = {
         let urlTextField = UITextField()
+        urlTextField.setContentCompressionResistancePriority(0, forAxis: UILayoutConstraintAxis.Horizontal)
```

Comment:
Don't need this anymore. We also shouldn't be setting the content compression resistance below 250 anyways.